lib/observer/known_locations.ex

```diff
@@ -0,0 +1,3 @@
+defmodule Observer.KnownLocations do
+  def seattle, do: Observer.create_from(47.6062, -122.3321, 0.370)
```

Comment:
Added the create_from helper function, because I wanted to allow lat/lon to be represented in degrees (as they are pretty much everywhere), but have one place that converts them to radians (which is needed in all of the relevant calculations)